app/package.json

```diff
@@ -5,11 +5,15 @@
   "scripts": {
     "build": "next build",
     "dev": "next dev",
+    "format": "prettier --write '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
+    "format-check": "prettier --check '**/*.{js,json,md,mdx,ts,tsx,scss,yaml,yml}'",
     "lint": "next lint",
     "start": "next start",
-    "test": "jest"
+    "test": "jest",
```

Comment:
Be careful that this PR change doesn't overwrite the one in #5 (`jest --ci --coverage`)